docs/design-decisions/ModelPromotion.md

```diff
@@ -0,0 +1,133 @@
+# Multi-Repository Model Development and Promotion Design
+
+## Overview
+
+This documentation outlines the design of an AI Model Factory consisting of two GitHub repositories: the Data Science / Prototyping Repository and the Model Factory / MLOps Repository. This design allows data scientists to focus on model development without the constraints of adhering to strict engineering practices, while ensuring that models are rigorously tested and deployed through automated processes.
```

Comment:
We need to emphasize in the beginning that the doc is more a proposal about how to use the repo. In the first paragraph it's not clear for the reader.

Reply:
Updated, thanks!